assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/style.scss

```diff
@@ -52,11 +52,12 @@ $border-width: 1px;
 			border-left: $border-width solid currentColor;
 			border-top: $border-width solid currentColor;
 			border-radius: $universal-border-radius 0 0 0;
+			border-color: $universal-border-light;
 			content: "";
 			display: block;
 			height: $universal-border-radius - $border-width;
 			margin-right: $gap-small;
-			opacity: 0.3;
+			opacity: 1;
```

Comment:
Are these values needed? 1 should be default.

Reply:
@mikejolley While testing the PR, there were some instances where opacity was overridden to 0.3 again. I added this manually to make sure it remains the same and also doesn't inherit the theme opacity styles.

Reply:
I think you should investigate where its coming from and fix there. It could be a mixin like `with-translucent-border`.

Reply:
@mikejolley Thank you for the suggestion! I have passed opacity argument in the `with-translucent-border` as it accepts opacity and assign it to the psuedo element: 

```css

@mixin with-translucent-border($border-width: 1px, $opacity: 0.3) {
	position: relative;

	&::after {
		border-style: solid;
		border-width: $border-width;
		bottom: 0;
		content: "";
		display: block;
		left: 0;
		opacity: $opacity;
		pointer-events: none;
		position: absolute;
		right: 0;
		top: 0;
	}
}
```